pkg/gatherers/workloads/workloads_info.go

```diff
@@ -63,37 +63,65 @@ func (g *Gatherer) GatherWorkloadInfo(ctx context.Context) ([]record.Record, []e
 	return gatherWorkloadInfo(ctx, gatherKubeClient.CoreV1(), gatherOpenShiftClient)
 }
 
-// nolint: funlen, gocyclo, gocritic
 func gatherWorkloadInfo(
 	ctx context.Context,
 	coreClient corev1client.CoreV1Interface,
 	imageClient imageclient.ImageV1Interface,
 ) ([]record.Record, []error) {
 	// load images as we find them
-	imageCh := make(chan string, workloadGatherPageSize)
-	imagesDoneCh := gatherWorkloadImageInfo(ctx, imageClient.Images(), imageCh)
+	imageCh, imagesDoneCh := gatherWorkloadImageInfo(ctx, imageClient.Images())
 
 	// load pods in order
 	start := time.Now()
+	limitReached, info, err := workloadInfo(ctx, coreClient, imageCh)
```

Comment:
The two comments above:
- `// load pods in order`
- `// load images as we find them`
seems to be at wrong place (as they were probably previously) so I would either move them or remove them.